pkgs/development/haskell-modules/configuration-tensorflow.nix

```diff
@@ -23,10 +23,7 @@ in
 {
   tensorflow-proto = doJailbreak (setTensorflowSourceRoot "tensorflow-proto" super.tensorflow-proto);
 
-  tensorflow = (setTensorflowSourceRoot "tensorflow" super.tensorflow).override {
-    # the "regular" Python package does not seem to include the binary library
-    libtensorflow = pkgs.libtensorflow-bin;
-  };
+  tensorflow = setTensorflowSourceRoot "tensorflow" super.tensorflow;
```

Comment:
> error: Package ‘tensorflow-0.2.0.1’ in /home/hexa/git/nixos/master/pkgs/development/haskell-modules/hackage-packages.nix:280221 is marked as broken, refusing to evaluate.

Reply:
Your change + explanation in this file seems sane. imo you can go ahead and ignore the error. If someone needs the package they are welcome to fix it/check if it works.